Header header
gbx_ros_bridge_msgs/BaseTime tSolution
float32 deltRSec